ddp-workspace/readvault.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+if [ "$1" == "" ]  ||  [ "$2" == "" ];
+then
+	echo "USAGE: readvault.sh vaultkey propertypath"
+	echo "EXAMPLE: readvault.sh secret/pepper/dev/v1/conf .data.gcp.serviceKey.token_uri"
+	exit 1
+fi
+vault read -format=json $1 | jq -r $2
```

Comment:
Just trying to keep code a little more readable within config.yml . Maybe there are other chunks in the config.yml that can be refactored to external scripts?